### PR TITLE
added inline code highlighting to match multiline style

### PR DESCRIPTION
**Pull request by jlloyd-widen**

### CODE REVIEW

File: _sass/_inline_code.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//this should apply only to inline code (i.e., `code`) not multiline code blocks
+
+.language-plaintext.highlighter-rouge  {
```
Comment (mskelton) [inline _sass/_inline_code.scss]: Make sure to import this in app.scss

Comment (jlloyd-widen) [reply _sass/_inline_code.scss]: Thank you! I'm a total noob to jekyll and sass, so I never would have gotten that. I just pushed a change.

**Comment by dbeg** [comment]
Kind of a nitpick, but the new CSS rule could probs be slammed into the existing syntax highlighting file: https://github.com/Widen/jekyll-theme-guardian/blob/master/_sass/_syntax-highlighting.scss (I'll defer to others' opinions tho).

Changes can be seen here: https://buildkite.yden.us/jekyll-theme-guardian/31/_site/style.html

image

Seems like maybe some L/R margin/padding on the new background might be more pleasing to the eye?

Also might be nice to match the border radius of the bigger code blocks?

```css
border-radius: 3px;
```

**Comment by jlloyd-widen** [comment]
> Kind of a nitpick, but the new CSS rule could probs be slammed into the existing syntax highlighting file: https://github.com/Widen/jekyll-theme-guardian/blob/master/_sass/_syntax-highlighting.scss (I'll defer to others' opinions tho).

happy to do this if @mskelton agrees.

I'm ready to push up a change that will add some padding on the left and right margins. Good idea!

File: _sass/_inline-code.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+//this should apply only to inline code (i.e., `code`) not multiline code blocks
+
+.language-plaintext.highlighter-rouge  {
+  background-color: #f8f8f8;
+  border: 1px solid #ccc;
+}
```
Comment (mskelton) [inline _sass/_inline-code.scss]: Some suggestions after looking at the preview site.

```suggestion
code.language-plaintext.highlighter-rouge  {
  background-color: #afb8c133;
  border-radius: 4px;
  padding: 0 4px;
}
```

**Review by mskelton — APPROVED** [review]

**Pull request merged**